tests/ui-testing/pages/ingestionPage.js

```
@@ -0,0 +1,38 @@
+    const fetchResponse = await fetch(
+      `${process.env.INGESTION_URL}/api/${orgId}/${streamName}/_json`,
+      {
+        method: "POST",
+        headers: headers,
+        body: JSON.stringify(logsdata),
+      }
+    );
+    try {
+      const response = await fetchResponse;
+      const contentType = response.headers.get("content-type");
+      if (contentType && contentType.includes("application/json")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)